### PR TITLE
About me

### PR DESCRIPTION
**Pull request by DmitriyKondaurov**
Home work  #1

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+    <section>
+        <h2>ДОПОЛНИТЕЛЬНАЯ ИНФОРМАЦИЯ</h2>
+        <p>Имею водительские права категории «А», «В», «В1» и опыт вождения автомобилем, мотоциклом.</p>
+    </section>
+</main>
+<footer>
+    <div class="rights">All rights reserved. Copyright</div>
+</footer>
+</body>
+</html>
```
Comment (spolischook) [inline index.html:139]: live empty line in the end of the file. Setup your IDE to add it on save by default

File: README.md [hunk 2]
```diff
@@ -1,2 +1,37 @@
-# Geekhub-FrontEnd-JS_2018
-Geekhub-FrontEnd+JS_2018
+## Welcome to GitHub Pages
```
Comment (spolischook) [inline README.md:1]: why you need this?

File: index.html [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+    </ul>
+</nav>
+<main>
+    <div class="main_photo" style="float: right; overflow: hidden; padding-top: 50px;">
+        <img src="img/My_photo.jpg" alt="My photo" width="180">
+    </div>
+    <h1 style="padding-top: 50px;">Кондауров Дмитрий Владимирович</h1>
+    <p>
+        1982 г.р. <br>
+        г. Черкассы, ул. Парижской Коммуны, 37 кв. 120 <br>
```
Comment (spolischook) [inline index.html:25]: it's not a good idea to share sensitive personal info world wide

**Review by spolischook — APPROVED** [review]
Ok except missing screenshot and few warnings

File: index.html (hunk 1 above)
Comment (DmitriyKondaurov) [reply index.html:139]: File | Settings | Editor | General --> Other --> Ensure blank line before end of file on Save
I chose this setting, but did not understand why we need this ?

**Pull request merged**